### ISSUE TITLE
newly started node stucks at block 500000

### ISSUE THREAD

**Issue by jagottsicher** [Crave-Project/Crave-NG#9]
I ran several masternodes with 2.5.0 and switched successfully to 2.5.1 on linux (ubuntu 14.04).
then I setup an additional node 2.5.1 and let it sync from the scratch. But I can't activate it due to the fact it seems to stuck processing block 500000.
```getinfo
{
    "version" : 2050100,
    "protocolversion" : 70002,
    "walletversion" : 61000,
    "balance" : 0.00000000,
    "zerocoinbalance" : 0.00000000,
    "blocks" : 500000,
    "timeoffset" : 0,
    "connections" : 0,
    "proxy" : "",
    "difficulty" : 47365.88651778,
    "testnet" : false,
    "moneysupply" : 18643866.35503282,
    "zCRAVEsupply" : {
        "1" : 73.00000000,
        "5" : 120.00000000,
        "10" : 410.00000000,
        "50" : 500.00000000,
        "100" : 1500.00000000,
        "500" : 4000.00000000,
        "1000" : 12000.00000000,
        "5000" : 40000.00000000,
        "total" : 58603.00000000
    },
    "keypoololdest" : 1558843915,
    "keypoolsize" : 1001,
    "paytxfee" : 0.00000000,
    "relayfee" : 0.00010000,
    "staking status" : "Staking Not Active",
    "errors" : ""
```
any ideas on it?

**Comment by SoundDrGenie** [comment]
There was an update to every wallet before everyone reach block 500 000. I forgot which version4:02 AM, May 26, 2019, Jens Schendel <notifications@github.com>:I ran several masternodes with 2.5.0 and switched successfully to 2.5.1 on linux (ubuntu 14.04).
then I setup an additional node 2.5.1 and let it sync from the scratch. But I can't activate it due to the fact it seems to stuck processing block 500000.
{
    "version" : 2050100,
    "protocolversion" : 70002,
    "walletversion" : 61000,
    "balance" : 0.00000000,
    "zerocoinbalance" : 0.00000000,
    "blocks" : 500000,
    "timeoffset" : 0,
    "connections" : 0,
    "proxy" : "",
    "difficulty" : 47365.88651778,
    "testnet" : false,
    "moneysupply" : 18643866.35503282,
    "zCRAVEsupply" : {
        "1" : 73.00000000,
        "5" : 120.00000000,
        "10" : 410.00000000,
        "50" : 500.00000000,
        "100" : 1500.00000000,
        "500" : 4000.00000000,
        "1000" : 12000.00000000,
        "5000" : 40000.00000000,
        "total" : 58603.00000000
    },
    "keypoololdest" : 1558843915,
    "keypoolsize" : 1001,
    "paytxfee" : 0.00000000,
    "relayfee" : 0.00010000,
    "staking status" : "Staking Not Active",
    "errors" : ""

any ideas on it?

—You are receiving this because you are subscribed to this thread.Reply to this email directly, view it on GitHub, or mute the thread.
-- Sent from Yandex.Mail for mobile

**Comment by jagottsicher** [comment]
Thanks, that is a good hint. On the releases site for version 2.5.1 here you can read: 

> MANDATORY UPDATE
> EVERY WALLET/NODE MUST BE UPGRADED BEFORE BLOCK 500000

So, my masternodes ran under 2.5.0 but delivered properly results, while my contol wallet has already 2.5.1, but I can not say for sure how long. Now after updating the masternodes to 2.5.1 (too late according to the note cited above) and created new address, priv key and transaction for it with an 2.5.1 wallet it should work as intended. 

at least the 2.5.1 masternode should be able to snyc higher than block 500000

**Comment by jagottsicher** [comment]
I saw, that I created address, priv key and transaction with an 2.5.0 controll wallet on windows OS. I can start over cause I have an 2.5.1 wallet also, but how can I now unlock exactly that 5000 carve from that not working wallet?

**Comment by jagottsicher** [comment]
So far so good. 
FYI:
I rsynced the content of one of my working nodes and excluded crave.conf.
then I restarted the stucking node with --reindex.
It already passed block 500000 so seems it worked. 

The other nodes have been restarted and after 24 hours they are delivering.

If the new mn also returns delivering blocks I'll update all 2.5.0 controller nodes to 2.5.1 and eventually copy the wallet.dat from the already working 2.5.1 controller wallet.

I can at any time dump all the private keys from the masternodes, I know all addressesand, transactions, but I am still not sure how to gain control over the funds (5000 deposit/stake) again, if I locked them in my controller node.

Anyway
